.xperior/testds/motr-single_tests.yaml

```diff
@@ -516,128 +517,128 @@ Tests:
-    sandbox  : /var/motr/sandbox.55sns-repair-motr-1n-1f
+    sandbox  : /var/motr/sandbox.61sns-repair-motr-1n-1f
     groupname: 03motr-single-node
     polltime : 30
     timeout  : 1800
 
   - id       : 56sns-repair-motr-mf
-    script   : sns_repair_motr_mf.sh
-    dir      : src/motr/st/utils/
+    script   : 'm0 run-st 62sns-repair-motr-mf'
+    dir      : src/scripts
     executor : Xperior::Executor::MotrTest
-    sandbox  : /var/motr/sandbox.56sns-repair-motr-mf
+    sandbox  : /var/motr/sandbox.62sns-repair-motr-mf
     groupname: 03motr-single-node
     polltime : 30
     timeout  : 1800
 
   - id       : 57sns-repair-motr-1k-1f
-    script   : sns_repair_motr_1k_1f.sh
-    dir      : src/motr/st/utils/
+    script   : 'm0 run-st 63sns-repair-motr-1k-1f'
+    dir      : src/scripts
     executor : Xperior::Executor::MotrTest
-    sandbox  : /var/motr/sandbox.57sns-repair-motr-1k-1f
+    sandbox  : /var/motr/sandbox.63sns-repair-motr-1k-1f
     groupname: 03motr-single-node
     polltime : 30
     timeout  : 600
 
   - id       : 58sns-repair-motr-ios-fail
-    script   : sns_repair_motr_ios_fail.sh
-    dir      : src/motr/st/utils/
+    script   : 'm0 run-st 64sns-repair-motr-ios-fail'
+    dir      : src/scripts
     executor : Xperior::Executor::MotrTest
-    sandbox  : /var/motr/sandbox.58sns-repair-motr-ios-fail
+    sandbox  : /var/motr/sandbox.64sns-repair-motr-ios-fail
     groupname: 03motr-single-node
     polltime : 30
     timeout  : 1800
 
   - id       : 59sns-repair-motr-abort
-    script   : sns_repair_motr_abort.sh
-    dir      : src/motr/st/utils/
+    script   : 'm0 run-st 65sns-repair-motr-abort'
+    dir      : src/scripts
     executor : Xperior::Executor::MotrTest
-    sandbox  : /var/motr/sandbox.59sns-repair-motr-abort
+    sandbox  : /var/motr/sandbox.65sns-repair-motr-abort
     groupname: 03motr-single-node
     polltime : 30
     timeout  : 1800
 
   - id       : 60sns-repair-motr-abort-quiesce
-    script   : sns_repair_motr_abort_quiesce.sh
-    dir      : src/motr/st/utils/
+    script   : 'm0 run-st 66sns-repair-motr-abort-quiesce'
+    dir      : src/scripts
     executor : Xperior::Executor::MotrTest
-    sandbox  : /var/motr/sandbox.60sns-repair-motr-abort-quiesce
+    sandbox  : /var/motr/sandbox.66sns-repair-motr-abort-quiesce
     groupname: 03motr-single-node
     polltime : 30
     timeout  : 1800
 
   - id       : 61spiel-multi-confd
-    script   : spiel_multiple_confd.sh
-    dir      : src/spiel/st/
+    script   : 'm0 run-st 08spiel-multiple-confd'
+    dir      : src/scripts
     executor : Xperior::Executor::MotrTest
-    sandbox  : /var/motr/sandbox.61spiel-multi-confd
+    sandbox  : /var/motr/sandbox.08spiel-multiple-confd
     groupname: 03motr-single-node
     polltime : 30
     timeout  : 1800
 
   - id       : 62dix-repair-st-lookup-insert
-    script   : dix_repair_st.sh -v lookup insert
-    dir      : src/dix/cm/st/
+    script   : 'm0 run-st 24motr-dix-repair-lookup-insert'
```

Comment:
The name in m0 run-st is "24motr-dix-repair-lookup-insert".
The name in .xperior is "62dix-repair-st-lookup-insert".

Can we make them unified?

Reply:
Updated.